File: exemplos/duas_linguagens/solution/PY3/lab00_main.py
#!/usr/bin/env python3

import lab00

import sys

l = 0
c = 0

while __name__ == '__main__':
    v = sys.stdin.read(1)
    if not v:
        break
    elif lab00.isBreakLine(v):
        l += 1
    else:
        c += 1

print("Caracteres: {0:d}".format(c));
print("Linhas:     {0:d}".format(l));

File: exemplos/duas_linguagens/solution/C/lab00_main.c
#include <stdio.h>

int isLineBreak(char c);

int main() {
  char v;
  int l = 0;
  int c = 0;

  while ((v = fgetc(stdin)) != EOF) {
    if (isLineBreak(v)) {
      l++;
    } else {
      c++;
    }
  }

  printf("Caracteres: %d\n", c);
  printf("Linhas:     %d\n", l);

  return 0;
}
